server/app/assets/javascripts/main.ts

```diff
@@ -262,8 +262,15 @@ function attachStopPropogationListenerOnFormButtons() {
 /**
  * Disables default browser behavior where pressing Enter on any input in a form
  * triggers form submission. See https://github.com/civiform/civiform/issues/3872
+ * Except when the ignore element is present, the default form action would work
+ * See https://github.com/civiform/civiform/issues/5464
  */
 function disableEnterToSubmitBehaviorOnForms() {
+  const ignoreElement = document.getElementById('cf-search-applications')
+  if (ignoreElement) {
+    return
+  }
```

Comment:
You're on the right track. Since you're looking for a specific ID this won't work well on forms with multiple input fields. 

Rather than tag each input we can just tag a specific form. I added an example below that adds a custom `data-` attribute to the form element. The `data-` attribute should not conflict with any existing or future libraries.

This way you just have to add that attribute to a form and the whole form will now work as browsers were intended to function by default.